src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -144,6 +117,15 @@ private void renumberColumns() {
 			c.setPos(i++);
 		}
 	}
+
+	public void setDefaultColumnEncodings() {
```

Comment:
because if a database's default encoding changes, its subordinate table's default encodings do *not* change.  I checked.